### PR TITLE
feat:implemented sum of numbers

### PR DESCRIPTION
**Pull request by manvithalva12**
implemented sum of given numbers

### CODE REVIEW

File: experiment.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function sum(numbers){
```
Comment (Rajharsh2) [inline experiment.js:1]: Please mention Jsdoc there.

**Review by Rajharsh2 — CHANGES_REQUESTED** [review]

File: experiment.js (hunk 1 above)
Comment (albin1616) [inline experiment.js:1]: change function nane

**Review by albin1616 — CHANGES_REQUESTED** [review]

File: experiment.js (hunk 1 above)
Comment (manvithalva12) [reply experiment.js:6]: resolved

**Review by Rajharsh2 — APPROVED** [review]
LGTM

**Review by albin1616 — APPROVED** [review]

**Pull request merged**